### PR TITLE
app note: Added new Xcore app note

### LINKED ISSUE #108: Add an Xcore app note
Per @bjhargrave's request, this issue tracks adding an app note on how to get Xcore to work with Bndtools.

### PR DESCRIPTION
**Pull request by jonpasski**
Shows how to get Xcore to work in Bndtools via the UI and command line.

Closes #108

Signed-off-by: Jon Passki <jon@passki.us>

### CODE REVIEW

File: _appnotes/xcore-bndtools.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
+---
+title: Getting Xcore to work in Bndtools
+summary: How to make an Xcore project work within Bndtools, both via the UI and command line.
+---
+
+[![Build Status](https://travis-ci.org/CoastalHacking/xcore-bndtools-example.svg?branch=master)](https://travis-ci.org/CoastalHacking/xcore-bndtools-example)
```
Comment (bjhargrave) [inline _appnotes/xcore-bndtools.md]: Not sure we should have the build status icon/link here. Can you remove?

File: _appnotes/xcore-bndtools.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+* `src/com.example.bndtools.xcore.model.*` Java files
+* `build.properties`
+* `plugin.properties`
+* `plugin.xml`
+
+Note: Xcore shouldn't add any additional libraries to the Xcore project build path.
+
+#### Unit Test
+
+Refer to the [Testing the Provider with %28Standard%29 JUnit](tutorial_base/340-junit.html)
```
Comment (bjhargrave) [inline _appnotes/xcore-bndtools.md]: I think the link needs to start with /
`(/tutorial_base/340-junit.html)`

File: _appnotes/xcore-bndtools.md (hunk 1 above)
Comment (jonpasski) [inline _appnotes/xcore-bndtools.md]: Yep, no problem.

File: _appnotes/xcore-bndtools.md (hunk 2 above)
Comment (jonpasski) [inline _appnotes/xcore-bndtools.md]: Good catch

**Comment by jonpasski** [comment]
Updated to include the feedback. How's it look now?

**Pull request merged**